lib/ui/platform_dispatcher.dart

```diff
@@ -417,6 +417,7 @@ class PlatformDispatcher {
         panDeltaY: packet.getFloat64(kStride * offset++, _kFakeHostEndian),
         scale: packet.getFloat64(kStride * offset++, _kFakeHostEndian),
         rotation: packet.getFloat64(kStride * offset++, _kFakeHostEndian),
+        preferredAction: PointerPreferredAction.values[packet.getInt64(kStride * offset++, _kFakeHostEndian)],
```

Comment:
it's not clear what "preferredAction" or `PointerPreferredAction` means in this context. How about `preferredStylusAuxilliaryAction` and `PointerPreferredStylusAuxilliaryAction`

Reply:
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8790504096978029009/+/u/test:_Host_Tests_for_host_debug_unopt/stdout
```
[0;32m[ RUN      ] [mShellTest.CanCorrectlySynthesizePointerPacket
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: RangeError (index): Invalid value: Not in inclusive range 0..3: 1731047719413088311
#0      _Array.[] (dart:core-patch/array.dart:10:36)
#1      PlatformDispatcher._unpackPointerDataPacket (dart:ui/platform_dispatcher.dart:420:55)
#2      PlatformDispatcher._dispatchPointerDataPacket (dart:ui/platform_dispatcher.dart:364:9)
#3      _dispatchPointerDataPacket (dart:ui/hooks.dart:91:31)
```

Reply:
I think its actually really confusing if both the actual event and the preferredAction are called PointerPreferredStylusAuxilliaryAction and PointerStylusAuxilliaryAction respectively,  I got confused just now between the two as Im trying to change the names because then the only difference them is the "preferred", and its pretty verbose

Reply:
How about StylusSecondaryGesture (double tap) + ConfiguredStylusAction (the preferred action)?